Automattic/woocommerce-payments
# change capture warning if merchant is eligible for ipp

Fixes #2921

#### Changes proposed in this Pull Request

Update capture warning to indicate in-person payments must be captured in 48 hours





If merchant is eligible for in-person payments, add a note to the "Issue an authorization on checkout, and capture later" setting: "The setting is not applied to In-Person Payments (please note that In-Person Payments should be captured within 2 days of authorization)."




![must-be-captured](https://user-images.githubusercontent.com/82826872/135053172-28122582-a369-4175-9973-1f78ffa24d67.png)

#### Testing instructions





* set `card_present_enabled` meta to 1 for the current store
* go to WooCommerce->Settings->Payments
* in "Transaction preferences" section, under "Issue an authorization on checkout, and capture later" field you should see a new help text: "Note: This setting is ignored by in-person payments. In-person payments authorization is valid for 48 hours."
* the text should not be present if in-person payments are not enabled.

-------------------


**Post merge**
